piet-svg/src/text.rs

```diff
@@ -60,11 +63,20 @@ pub struct TextLayoutBuilder;
 
 impl piet::TextLayoutBuilder for TextLayoutBuilder {
     type Out = TextLayout;
+    type Font = Font;
 
     fn alignment(self, _alignment: piet::TextAlignment) -> Self {
         self
     }
 
+    fn add_attribute(
+        self,
+        _range: impl RangeBounds<usize>,
+        _attribute: impl Into<TextAttribute<Self::Font>>,
+    ) -> Self {
+        self
```

Comment:
Should there be at least a TODO here indicating that it's unimplemented functionality?

Reply:
the whole svg text API is unimplemented; I don't think you can even get a `TextLayoutBuilder` without panicking.